### DISCUSSION TITLE
Is it possible to insert objects? Like Word and Excel documents

### DISCUSSION THREAD

**Discussion by GuidoPreite** [ShapeCrawler/ShapeCrawler#622] [Q&A] [answered]
I need to create a PowerPoint file (using C#) but I need to insert an object to the first slide.

Screenshot on how it is done inside PowerPoint:
![image](https://github.com/ShapeCrawler/ShapeCrawler/assets/43754988/7b9d5a49-95b2-4088-9629-a980c820f37f)

I googled but didn't find an example, is it possible to insert objects using ShapeCrawler?

By the way very nice library, I am sure will use it in the future for "normal" PowerPoint files.

**Comment by AdamShakhabov** [comment, marked as answer]
Hello @GuidoPreite. Thank you for your feedback👍

No, this feature is not available yet.
